crates/core/tedge/src/cli/connect/command.rs

```diff
@@ -74,8 +74,8 @@ impl Command for ConnectCommand {
             let br_config = self.bridge_config(&config)?;
             if self.check_if_bridge_exists(&br_config) {
                 return match self.check_connection(&config) {
-                    Ok(DeviceStatus::AlreadyExists) => {
-                        let cloud = br_config.cloud_name.clone();
+                    Ok(_) => {
+                        let cloud = br_config.cloud_name;
```

Comment:
@rina23q I believe this check against a particular variant was done for a reason, could you have a look at this?

Reply:
Thanks for catching it! Yes, Pradeep and I worked on the fix of this condition by #861. 
We have a variant `Ok(DeviceStatus::Unknown)`, which we should not print "successful" in the output.
With the variant, the connection check has publish a message successfully, however, we didn't get the expected response by cloud for some reason. That's why we can't print "successful".